eclipse-theia/theia
# repo: update bundled builtins

#### What it does


Fixes: https://github.com/eclipse-theia/theia/issues/9985.

The pull-request updates the list of builtins (bundled plugins) which are used in the repository for test purposes (ultimately the bundled plugins used by both the _browser_ and _electron_ examples). It is important to update the builtins following a default api bump (https://github.com/eclipse-theia/theia/pull/9959) so we can identify issues more easily and work towards increased api compatibility.

The update includes the use of the builtins-extension-pack (which can be resolved at builtime to the latest supported version based on the api), and the use of an exclude list (to exclude the list of plugins we do not want as part of the application).

**Known Issues**:

- [x] https://github.com/eclipse-theia/theia/issues/8662 by `typescript-language-features` on startup.

#### How to test


- perform `yarn download:plugins` - the pack should be installed and valid plugins should be resolved.
- start the application.
- identify any potential issues or incompatibilities.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
